kpi/utils/redis_helper.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+import re
+
+
+class RedisHelper(object):
+    """
+    Redis's helper.
+
+    Mimics dj_database_url
+
+    """
+
+    @staticmethod
+    def config(default=None):
+        """
+        :return: dict
+        """
+
+        try:
+            redis_connection_url = os.getenv("REDIS_SESSION_URL", default)
+            match = re.match(r"redis://(:(?P<password>[^@]*)@)?(?P<host>[^:]+):(?P<port>\d+)(/(?P<index>\d+))?",
+                             redis_connection_url)
+            if not match:
+                raise Exception()
```

Comment:
Yes, let's raise an exception here (and not catch it). Let's use `ImproperlyConfigured` and the text already included below (`"Could not parse Redis session URL. Please verify 'REDIS_SESSION_URL' value"`)